main.py

```diff
@@ -253,7 +256,15 @@ def event_create(self, event):
         error = False
 
         try:
-            box = Box(event.content['data'], event.content['namespace'])
```

Comment:
Too much code inside the `try/except`, please try to make the try/except block as small as possible.